packages/checkout-storefront/src/lib/translations/en-US.json

```diff
@@ -11,6 +11,7 @@
   "account": "Account",
   "signOut": "Sign Out",
   "signIn": "Sign In",
+  "checkoutFinalizeEmailInvalidError": "Please provide valid email before finalizing checkout",
```

Comment:
This is needed because we generate those keys based on mutation name `checkoutFinalize` + field name `Email` + error type `Invalid` + `Error` -> and we don't type them anywhere in the code manually. So the fact these two are the same right now is only because of the fact that I didn't come up with a better messages, but they're needed